src/sbin/livemedia-creator

```diff
@@ -238,8 +238,8 @@ def default_image_name(compression, basename):
 
 if __name__ == '__main__':
```

Comment:
How about we just move all this into a main() function and call that to keep from polluting the global namespace.